README.md

```diff
@@ -5,11 +5,1119 @@
+
+```ts
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.name({ alias: 'handle' })
+  },
+})
+```
+
+```gql
+type User {
+  handle: String
+}
+```
+
+#### `type` Option
+
+Only available for relational model fields. Use `type` to change the name of the projected field type. This is useful when the model that the field relates to has been projected onto a differently named `Object`.
+
+```ts
+objectType({
+  name: 'Article',
+  definition(t) {
+    t.model('Post').title()
+  },
+})
+
+objectType({
+  name: 'User',
+  definition(t) {
+    t.model.posts({ alias: 'articles', type: 'Article' })
+  },
+})
+```
+
+```gql
+type Article {
+  title: String
+}
+
+type User {
+  articles: [Article]
+}
+```
+
+#### `ordering` `pagination` `filtering` Options
+
+Only available for list type model fields. Please refer to TODO for details.
+
+### `t.crud`
+
+`t.crud` is for exposing operations against your projected models. It is only available inside `Query` and `Mutation` definitions. `t.crud` properties are called "Operation Publishers". They accept configuration allowing you to control how the operation is seen through your GraphQL schema.
+
+`t.crud` reflects a subset of [Photon](https://photonjs.prisma.io)'s capabilities. For every one model in your Prisma schema there are ten corresponding Operation Publishers.
+
+```prisma
+model User {
+  id Int @id
+}
+```
+
+```ts
+queryType({
+  definition(t) {
+    t.crud.user()
+    t.crud.users()
+  },
+})
+
+mutationType({
+  definition(t) {
+    t.crud.createOneUser()
+    t.crud.udpateOneUser()
+    t.crud.upsertOneUser()
+    t.crud.deleteOneUser()
+
+    t.crud.createManyUser()
+    t.crud.updateManyUser()
+    t.crud.upsertManyUser()
+    t.crud.deleteManyUser()
+  },
+})
+```
+
+The following Operation Publisher docs follow this format:
+TODO revise this it is ugly
+
+```
+<overview>
+<contributions to your GraphQL schema>
+<example>
+  <GraphQL client operations>
+  <GraphQL schema>
+  <Nexus/Nexus-Prisma type defs>
+  <Prisma schema model defs>
+```
+
+#### One-Create Operation
+
+Allow clients to create records of the respective model.
+
+Relation fields may be connected with an existing record or a sub-create may be inlined. If the relation is a `List` then multiple connections or sub-creates are permitted.
+
+Each instance of this operation contributions to GraphQL Schema:
+
+- `×1` `InputObject` `<ModelName>CreateInput`
+- `×N` `InputObject` per relational field.
+
+  This is a container for the sub-create and connect (below). Cannot be amortized becuase sub-create cannot.
+
+- `×N` `InputObject` per relational field create.
+
+  Cannot be amoritized because of specialized shape excluding field where supplying relation of type of parent `Object` being created would normally be. Instead this relation being created will be intuitively connected to the `Object` being directly created.
+
+- `×N (A)` `InputObject` `<ModelName>WhereUniqueInput` per relational field connect.
+
+Example:
```

Comment:
> add at one sentence or a short description before each code block to explain what this is/where it comes from

The reference examples follow a repeated pattern that aims to be lean. The goal is illustrating the information across 4 layers related to that operation. It is abstract with regards to how an app is formed.

Maybe a better heading here (than Example) would be Illustration.

> I'd lead with it since it sets the stage for everything that follows?

It's in order of the stack layers, but we could reverse. Right now its in a literate programming style order. Beginning with the result, working backward to the implementation. This approach allows one to ask "How is that working?, How is that working?, ..." vs "Why this/what happens next?, Why this/what happens next? ...". 

Let's revisit this once the content is done and see how it feels overall. Not sure yet.